types/presto-client/index.d.ts

```diff
@@ -321,6 +321,23 @@ export interface QueryOptions {
 
 export class Client {
     constructor(options: ClientOptions);
+
+    userAgent: string;
+    host: string;
+    port: number;
+    user?: string;
```

Comment:
I think we can make this always set:

```suggestion
    user: string
```

where if you don't set a username, then presto/trino will throw an error when you run a query, and also that unless you have done some weird environment variable shenenanigans, then `process.env.USER` should always be the fallback if not specified:

https://github.com/tagomoris/presto-client-node/blob/893a6d07cd6a3267b2c84699dee8f8872faaedcd/lib/presto-client/index.js#L36

Reply:
Thanks. Updated.